libbeat/testing/integration/run_beat.go

```diff
@@ -126,7 +126,7 @@ type RunBeatOptions struct {
 	KeepRunning bool
 }
 
-// Runs a Beat binary with the given config and args.
+// RunBeat a Beat binary with the given config and args.
```

Comment:
```suggestion
// RunBeat runs a Beat binary with the given config and args.
```

Reply:
done in https://github.com/elastic/beats/pull/42914/commits/9cfe7de7438024cda539b7c25a965118f41bcece